test/runtests.jl

```diff
@@ -75,6 +75,12 @@ if !Sys.iswindows()
     @safetestset "Hybrid - dss opt" begin @time include("Operators/hybrid/dss_opt.jl") end
     @safetestset "Hybrid - opt" begin @time include("Operators/hybrid/opt.jl") end
 
+    @safetestset "MatrixFields - BandMatrixRow" begin @time include("MatrixFields/band_matrix_row.jl") end
+    @safetestset "MatrixFields - rmul_with_projection" begin @time include("MatrixFields/rmul_with_projection.jl") end
+    @safetestset "MatrixFields - field2arrays" begin @time include("MatrixFields/field2arrays.jl") end
```

Comment:
Let's add these tests separately to buildkite. Please see https://github.com/CliMA/ClimaCore.jl/pull/1340 for the direction I'm hoping runtests.jl moves to.